lte/cloud/go/services/policydb/obsidian/handlers/policy_handlers.go

```diff
@@ -254,6 +254,25 @@ func CreateRule(c echo.Context) error {
 		return obsidian.HttpError(err, http.StatusBadRequest)
 	}
 
+	if rule.FlowList != nil {
```

Comment:
this nilcheck is redundant: https://play.golang.org/p/iTq3xOqr494

for loop will no-op when list is nil

Reply:
nifty